tests/vmi_multus_test.go

```diff
@@ -698,6 +698,25 @@ var _ = Describe("[Serial]SRIOV", func() {
 			}
 		}
 
+		It("[Serial]should create virt-launcher pod with CAP_SYS_RESOURCE capability", func() {
+			tests.EnableFeatureGate(virtconfig.SRIOVLiveMigrationGate)
+			// Using UpdateKubeVirtConfigValueAndWait expects a test to run in serial mode
+			defer tests.UpdateKubeVirtConfigValueAndWait(tests.KubeVirtDefaultConfig)
+
+			vmi := getSriovVmi([]string{sriovnet1}, defaultCloudInitNetworkData())
+			vmi = startVmi(vmi)
+			vmi = waitVmi(vmi)
+
+			By("Looking up for VMI virt-launcher pod using VMI's label")
+			virtLauncherPod := tests.GetRunningPodByVirtualMachineInstance(vmi, tests.NamespaceTestDefault)
```

Comment:
Please take the namespace from `vmi.Namespace`.

Reply:
Done

Reply:
Done